libnetwork/drivers/bridge/port_mapping_linux.go

```diff
@@ -40,21 +48,33 @@ type portBinding struct {
 	stopProxy func() error
 }
 
+// childPortBinding is pb.PortBinding, with the host address the daemon
```

Comment:
Yes, that'll probably be the way we want to do it when we get to nftables, depending on the approach we take.

The interface/composition will need a bit of thought, because they're not independent (as I think you've seen in the code highlighted in this comment) - docker-proxy and iptables rules need to use a host address determined by rootlesskit.